crates/node/src/dry_run_tx.rs

```diff
@@ -37,7 +37,7 @@ where
     let gas_scale = parameters::get_gas_scale(&state)?;
 
     // Wrapper dry run to allow estimating the gas cost of a transaction
-    let (wrapper_hash, extended_tx_result, tx_gas_meter) =
+    let (wrapper_hash, extended_tx_result, tx_gas_meter, gas_used) =
```

Comment:
The modifications to this file are just temporary to fix a bug for which we were not tracking gas correctly. A better implementation is provided in PR #3758  which is based on this one.